modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -49,3 +49,22 @@ You can force the reconciliation by making any minor change to the DPA itself.
 ----
 $ oc -n openshift-adp get secret/oadp-<bsl_name>-<bsl_provider>-registry-secret -o json | jq -r '.data'
 ----
+
+
+[id="openshift-adp-controller-manager-seg-fault_{context}"]
+== OpenShift ADP Controller segmentation fault
+
+If you configure a DPA with both `cloudstorage` and `restic` enabled, the `openshift-adp-controller-manager` pod crashes and restarts indefinitely until the pod fails with a crash loop segmentation fault.
+
+You can have either `velero` or `cloudstorage` defined, as they are mutually exclusive fields.
```

Comment:
"as" -> "because" per IBM style guide. But I wouldn't hold up the merge over just this.

![image](https://github.com/openshift/openshift-docs/assets/7443887/72012a48-c348-479c-8191-5de32c6833a5)